extensions.json

```
@@ -1256,6 +1261,11 @@
       "version": "0.0.1",
       "checkout": "f9fef4e2e6867f9850809f9a7f54e700e17fc7d5"
     },
+    {
+      "id": "RandomChance.logstash",
+      "repository": "https://github.com/randomchance/vscode-logstash-configuration-syntax",
+      "version": "0.0.4"
```

This extension repository doesn't provide any `.vsix` GitHub Releases or suitable release tags. In that case, please don't pin a specific version:

```suggestion
      "repository": "https://github.com/randomchance/vscode-logstash-configuration-syntax"
```

(Hint: Running `node add-extension https://github.com/randomchance/vscode-logstash-configuration-syntax --checkout` can automatically detect this and should generate the expected config.)